### PR TITLE
[MS] Updated file import

### LINKED ISSUE #7063: Manage folder import in "Import" modal
At the moment we can import folder only by drag and drop, we need to be able to do the same in the "Import" modal

### PR DESCRIPTION
**Pull request by Max-7**
Closes #7063 

- [X] Keep changes in the pull request as small as possible
- [X] Ensure the commit history is sanitized
- [X] Give a meaningful title to your PR
- [X] Describe your changes
- [X] Link any related issue in the description

### CODE REVIEW

File: client/src/components/files/FileImportPopover.vue [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        <ion-label class="body item-label">
+          {{ 'Files' }}
+        </ion-label>
+      </ion-item>
+      <ion-item
+        class="main-list__item"
+        @click="onOptionClick(ImportType.Folder)"
+      >
+        <ion-label class="body item-label">
+          {{ 'Folders' }}
```
Comment (NicoTuxx) [inline client/src/components/files/FileImportPopover.vue:19]: should be translated

File: client/src/components/files/index.ts [hunk 2]
```diff
@@ -2,22 +2,24 @@
 
 import FileCard from '@/components/files/FileCard.vue';
 import FileCardImporting from '@/components/files/FileCardImporting.vue';
-import FileDropZone from '@/components/files/FileDropZone.vue';
 import FileGridDisplay from '@/components/files/FileGridDisplay.vue';
 import FileImport from '@/components/files/FileImport.vue';
```
Comment (NicoTuxx) [inline client/src/components/files/index.ts:6]: remove this line

**Review by Ironicbay — APPROVED** [review]
LGTM

File: client/src/locales/en-US.json [hunk 3]
```diff
@@ -463,7 +465,8 @@
             },
             "noImportInProgress": "No import in progress",
             "noImportDone": "No import done",
-            "noImportFailed": "No import failed"
+            "noImportFailed": "No import failed",
+            "dropInstructions": "Drag & drop your files here to import them"
```
Comment (Ironicbay) [inline client/src/locales/en-US.json:469]:
```suggestion
            "dropInstructions": "Drag & drop your files to import them"
```

**Review by Ironicbay — CHANGES_REQUESTED** [review]
It's impossible to drag & drop in an empty workspace

File: client/src/components/files/FileDropZone.vue [hunk 4]
```diff
@@ -5,21 +5,52 @@
...
+import { computed, onMounted, onUnmounted, ref } from 'vue';
+
+defineExpose({
+  reset,
+});
+
+const props = defineProps<{
+  currentPath: FsPath;
+  disabled?: boolean;
+  showDropMessage?: boolean;
```
Comment (Ironicbay) [inline client/src/components/files/FileDropZone.vue:42]: How useful is the `showDropMessage` prop ? It's set to `true` in all cases.

**Review by Ironicbay — CHANGES_REQUESTED** [review]
When importing a folder, it would be more natural to display the folder itself being imported in the `FileUploadMenu`, and to have a count of the files being imported inside (`m / n items imported`) next to the folder progress bar.

Related issue: https://github.com/Scille/parsec-cloud/issues/7134

File: client/src/components/files/FileDropZone.vue (hunk 4 above)
Comment (Max-7) [reply client/src/components/files/FileDropZone.vue:42]: Nop, not when dropping on a file/folder. When it's set, it's indeed set to true. But there are cases when it's not set.

**Review by Ironicbay — APPROVED** [review]

**Pull request merged**